fuzz-afl/fuzz_targets/afl_parse_elf.rs

```diff
@@ -0,0 +1,26 @@
+#[macro_use]
+extern crate afl;
+
+fn main() {
+    fuzz!(|data: &[u8]| {
+        if let Ok(elf) = goblin::elf::Elf::parse(data) {
+            for section_header in &elf.section_headers {
+                let _ = elf.shdr_strtab.get(section_header.sh_name);
```

Comment:
Maybe use [`black_box()`](https://doc.rust-lang.org/std/hint/fn.black_box.html) to avoid the compiler optimizing everything away?

Reply:
![goblin](https://user-images.githubusercontent.com/38811376/210258393-977d7216-ca80-4e54-995b-e94b134a0c5b.png)
I build with `cargo afl build --release` this line is still here, according to IDA. You helped me to notice this:

    warning: use of deprecated associated function `goblin::strtab::Strtab::<'a>::get`: Bad performance, use get_at() instead

 I think we should change `get` method to `get_at`.